common/src/main/java/co/mobileaction/example/common/dto/UserCrawlRequestDto.java

```diff
@@ -0,0 +1,8 @@
+package co.mobileaction.example.dto;
+
+import lombok.*;
+
+@Data @NoArgsConstructor @AllArgsConstructor
```

Comment:
These should be on top of each other, not on one single line
```suggestion
@Data 
@NoArgsConstructor 
@AllArgsConstructor
```